src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -411,6 +421,64 @@ def visit_span(self, span: Span):
             )
 
 
+class RenderBlockingAssetSpanDetector(PerformanceDetector):
+    __slots__ = ("stored_issues", "fcp", "transaction_start")
+
+    settings_key = DetectorType.RENDER_BLOCKING_ASSET_SPAN
+
+    def init(self, event: Event):
+        self.stored_issues = {}
+        self.transaction_start = timedelta(seconds=event.get("transaction_start", 0))
+        self.fcp = None
+
+        # Only concern ourselves with transactions where the FCP is within the
+        # range we care about.
+        fcp_hash = event.get("measurements", {}).get("fcp", {})
+        if "value" in fcp_hash and ("unit" not in fcp_hash or fcp_hash["unit"] == "millisecond"):
```

Comment:
I couldn't find anywhere where we're dealing with different units for measurement values. The frontend SDK, which is the only thing generating these resource spans, uses `millisecond`.

Reply:
Better to be more restrictive, the output inside our metrics will show us whether it was somehow too restrictive, and we can watch it over time.